draft-ietf-rats-corim.md

```diff
@@ -2097,19 +2097,15 @@ The comparison algorithm performed depends on the value of the codepoint being c
 
 [^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/203
 
-If the `measurement-values-map` value has an associated CBOR tag.
-The comparison algorithm should comprehend the structure identified by the CBOR tag.
+If the `measurement-values-map` value has an associated CBOR tag, the comparison algorithm should comprehend the structure identified by the CBOR tag.
 
-If the stateful environment `measurement-values-map` value is tagged with a CBOR tag {{-cbor}} then the Verifier MUST use the comparison algorithm associated with that tag.
+If the Verifier does not recognize a CBOR tag value then the value MUST NOT match.
 
-If the value is not tagged then the Verifier MUST use the comparison algorithm associated with the `measurement-values-map` codepoint for the entry.
+Note: CBOR tags are useful for discriminating values amongst alternates, but the comparison of tagged values is still determined by the codepoint they appear under. It is recommended but not required to compare values with a specific CBOR tag the same way across codepoints. For this reason, it is recommended to specify a default comparison algorithm with the CBOR tag's registration.
```

Comment:
```suggestion
The CBOR tag's registration specification SHOULD specify a comparison algorithm if not defined in this or other specification.

Note: The comparison algorithm of CBOR tagged values is determined by the key (i.e., codepoint) they appear under.
It is recommended, but not required, that comparison algorithms involving the same CBOR tag process the tagged information similarly across the different codepoints.
```

Reply:
I would correct the `suggested` edit on New Line  2107 to:

The CBOR tag's registration specification SHOULD specify a comparison algorithm either in this specification or in an equivalent profile document of this specification

Reply:
>this specification or in an equivalent profile

I was leaving the door open for other I-Ds or even other standards bodies could define extensions to measurement-values-map or triples-map that result in the need for a comparison algorithm definition. The IANA experts are in charge of overseeing that the extensions are aligned with base corim and other standard extensions. A profile can't define a "standard" extension / comparison algorithm (based on an IANA registry), but it can define non-standard ones using CBOR tags' vendor specific range.

Reply:
I don't know what vendor specific range you're talking about. https://www.iana.org/assignments/cbor-tags/cbor-tags.xhtml has 3 classes of ranges, and none of them suggest that a tag number can be reinterpreted in a vendor-specific way.

Reply:
> I don't know what vendor specific range you're talking about. https://www.iana.org/assignments/cbor-tags/cbor-tags.xhtml has 3 classes of ranges, and none of them suggest that a tag number can be reinterpreted in a vendor-specific way.

The semantics for allocating from the first come first served range is the definition of what is behind the tag doesn't have to be in a standard specification. The requester can just say what it is. This process makes it effectively a vendor specific / defined CBOR tag.